flloat/base.py

```diff
@@ -72,60 +72,29 @@ def find_labels(self) -> Set[AtomSymbol]:
         return {self.s}
 
 
-class QuotedFormula(Hashable):
+class QuotedFormula(Wrapper):
```

Comment:
With the new class `helpers.Wrapper`, which contains the common logic of wrappers, `QuotedFormula` can be simplified a lot.